mermithid/processors/misc/TritiumAndEfficiencyBinner.py

```diff
@@ -0,0 +1,152 @@
+
+        # Put it all in a dictionary
+        self.results = {self.output_bin_variable: [], 'N': [], 'bin_efficiencies': [], 'bin_efficiency_errors': [], 'event_efficiencies': [], 'event_efficiency_errors': []}
+        self.results['N'] = N
+        self.results[self.output_bin_variable] = self.bin_centers
+        self.results['bin_efficiencies'] = self.bin_efficiencies_normed
+        self.results['bin_efficiency_errors'] = self.bin_efficiency_errors_normed
+        self.results['event_efficiencies'] = self.event_efficiencies_normed
+        self.results['event_efficiency_errors'] = self.event_efficiency_errors_normed
+
+        return True
+
+    def EfficiencyAssignment(self, f_bin_centers, f_bins = None, integrate_bin_width = False):
+        fss_frequencies = self.efficiency_file_content['frequencies']
+        fss_efficiencies = self.efficiency_file_content['eff interp with slope correction']
+        fss_efficiency_errors = self.efficiency_file_content['error interp with slope correction']
+        efficiency_interpolation = scipy.interpolate.interp1d(fss_frequencies, fss_efficiencies, bounds_error=False, fill_value=0)
+        efficiency_error_interpolation_lower = scipy.interpolate.interp1d(fss_frequencies, fss_efficiency_errors[0], bounds_error=False, fill_value=1)
+        efficiency_error_interpolation_upper = scipy.interpolate.interp1d(fss_frequencies, fss_efficiency_errors[1], bounds_error=False, fill_value=1)
+        if integrate_bin_width == False:
+            # FOI means frequency of interest
+            logger.info('Not integrating efficiencies')
+            FOI_efficiencies = efficiency_interpolation(f_bin_centers)
+            FOI_efficiency_lower_errors = efficiency_error_interpolation_lower(f_bin_centers)
+            FOI_efficiency_upper_errors = efficiency_error_interpolation_upper(f_bin_centers)
+        else:
+            logger.info('Integrating efficiencies')
+            number_of_bins = len(f_bins)-1
+            FOI_efficiencies = np.zeros(number_of_bins)
+            FOI_efficiency_lower_errors = np.zeros(number_of_bins)
+            FOI_efficiency_upper_errors = np.zeros(number_of_bins)
+            for i in range(number_of_bins):
+                FOI_efficiencies[i] = scipy.integrate.quad(efficiency_interpolation, f_bins[i], f_bins[i+1])[0]/(f_bins[i+1]-f_bins[i])
+                FOI_efficiency_lower_errors[i] = scipy.integrate.quad(efficiency_error_interpolation_lower, f_bins[i], f_bins[i+1])[0]/(f_bins[i+1]-f_bins[i])
+                FOI_efficiency_upper_errors[i] = scipy.integrate.quad(efficiency_error_interpolation_upper, f_bins[i], f_bins[i+1])[0]/(f_bins[i+1]-f_bins[i])
+        # to do: figure out how to deal with the situation where frequencies have been slope/power corrected
+        return FOI_efficiencies, [FOI_efficiency_lower_errors, FOI_efficiency_upper_errors]
+
+    def GetEfficiencyFileContent(self):
```

Comment:
There is nothing in the InternalConfigure that can catch this false.
Consider adding a 
```python
if not self.efficiency_file_content = self.GetEfficiencyFileContent():
    logger.error("Failed reading efficiency file")
    return False
```
in the InternalConfigure